dashboard-app/src/dashboard/ByTime/index.tsx

```diff
@@ -84,10 +67,10 @@ const ByTime: FunctionComponent<RouteComponentProps> = (props) => {
   }, [tableData]);
 
   const downloadAsCsv = useCallback(() => {
-    downloadCsv({ aggData, fromDate, toDate, setErrors, fileName: 'by_activity', unit });
-  }, [aggData, fromDate, toDate, unit]);
+    downloadCsv({ aggData: data, fromDate, toDate, setErrors, fileName: 'by_activity', unit });
```

Comment:
I think we should change all data manipulation functions to take a `data` arg rather than `aggData`